lib/vmfloaty.rb

```diff
@@ -256,10 +272,13 @@ def run # rubocop:disable Metrics/AbcSize
         end
 
         unless successes.empty?
-          FloatyLogger.info unless failures.empty?
-          puts 'Scheduled the following VMs for deletion:'
-          successes.each do |hostname|
-            puts "- #{hostname}"
+          if options.json
+            puts successes.to_json
+          else
+            puts 'Scheduled the following VMs for deletion:'
+            successes.each do |hostname|
+              puts "- #{hostname}"
+            end
```

Comment:
I think we can make this better by first building up the output string, and then only printing once rather than new `puts` for each hostname.